Gemfile.lock

```diff
@@ -467,7 +491,7 @@ GEM
     zeitwerk (2.6.0)
 
 PLATFORMS
-  ruby
+  arm64-darwin-21
```

Comment:
:-o this breaks CI. Any chance to revert?

Reply:
yep, sorry!

Reply:
I had some environment issues that were causing more changes to the Gemfile than necessary, but I think I've gotten it straightened out now to only update the gems necessary for the webauthn change